RFC.md

```diff
@@ -0,0 +1,393 @@
+  Standard Multihash Table.
+- `<digest-lengt>` is a decimal number storing the length of
+  `<digest-value>`, in bytes.
+- `<digest-value-hex>` is the hash function digest value, hex-encoded.
+- `.` is a delimiter for the values.
+
+This representation SHOULD NOT be used to transmit, store, or embed a
+multihash value. It SHOULD only be used for debugging.
+
+Even when creating human-oriented string identifiers, it is strongly
+RECOMMENDED to use the packed representation of multihash, possibly
+encoded in a convenient base. This ensures the whole multihash value
+is as compact as it can be, and does not accidentally impose other
+requirements on the transmission of the value. It is much easier for
+systems to deal with a binary value that can be easily encoded in a
+variety of bases. Using the Explanation Representation for storage,
+transmission, embedding in other identifiers, or anything other
+than debugging, defeats the purpose of multihash.
+
+### MSB Unsigned Varints - muvints
+
+Multihash uses muvints, Most-significant-bit Unsigned Variable
+INTegerS. These are in use by other multiformats. Their definition
+is summarized here for completeness.
+
+Unsigned: muvints are unsigned integers. There is no need for
+  distinguishing negative integers.
+
+Varints: muvints are variable integers, with no limit.
+
+MSB continuation: muvints use the Most Significant Bit of every byte
+  to represent a continuation bit. This type of varint is optimized
+  for space and reads of small numbers, not for reads of very large
+  numbers (128-bit ints and beyond).
+
+Little-Endian: muvints are based on Protocol Buffers varints, and
+  are thus little-endian, meaning the least significant bytes are
+  encoded first.
+
+Examples:
+
+  # decimal   muvint bytes
+  127         01111111
+  128         10000000 00000001
+  256         10000000 00000010
+  1024        10000000 00001000
+  16384       10000000 10000000 00000001
+
+## Encoding multihashes
+
+Multihashes are designed to be used as values in a variety of
+mediums. They will often need to be encoded in other bases.
+In particular, multihash endeavors to keep the function code
+and length in the same base encoding as the digest value, which
+is important to many applications that must treat hash digest
+values opaquely, or that may have base encoding restrictions.
+
+Binary: It is RECOMMENDED that multihash values are stored
+  and transmitted on the wire as binary packed values wherever
+  possible. This will ensure the hash digests take up as little
+  space as possible.
+
+Copiable: It is RECOMMENDED that multihash values are displayed
+  to users in a "copiable" form, that is in a form easy to select,
+  copy, and paste, which typically means in base16, base32, base58.
+
+Multibase: Multihash pairs well with Multibase, a standard for
+  self-describing base encodings. This way, a multihash value can
+  be stored, transmitted, or displayed in any base without any
+  ambiguity.
+
+
+## Considerations
```

Comment:
Another considerations section could be about the use of multihashes in hostnames and URLs.